app/react/App/scss/modules/_settings-metadata.scss

```diff
@@ -139,6 +139,7 @@
 
 .property-description-right {
   @include tooltip('right');
+  z-index: 9999;
```

Comment:
this will only fix the problem for tooltips going right, probably you want to add that z-index to all tooltips.

Reply:
done